main/options/options.cc

```diff
@@ -598,10 +600,16 @@ void readOptions(Options &opts, int argc, char *argv[],
         }
         opts.disableWatchman = raw["disable-watchman"].as<bool>();
         opts.watchmanPath = raw["watchman-path"].as<string>();
+
+        // Certain features only need certain passes
         if ((opts.print.Autogen.enabled || opts.print.AutogenMsgPack.enabled || opts.print.AutogenClasslist.enabled) &&
-            (opts.stopAfterPhase != Phase::NAMER)) {
+            (opts.stopAfterPhase < Phase::NAMER)) {
```

Comment:
I don't think this necessarily does what you think. Also I imagine using resolution may change the autogen outputs in other ways.

Note that autogen already affects the parts of the pipeline that are executed in realmain: https://github.com/sorbet/sorbet/blob/master/main/realmain.cc#L403-L427

Reply:
+1, I think it's worth keeping it more specific and doing a `!=` check.

Reply:
So I did this b/c I assumed that `-p autogen-subclasses` would need `--stop-after=resolver` (rather than `namer`, and I didn't want to add another Sorbet invocation to `SorbetAutogen`) but after further testing that appears to not be the case.